qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/create/CreateOfferInput.groovy

```diff
@@ -9,4 +13,22 @@ package life.qbic.portal.portlet.offers.create
  */
 interface CreateOfferInput {
 
+
+    /**
+     * Creates a new quotation based on the given TOMATOID
+     * @param tomatoId makes it possible to identify the quotation
+     * @since 1.0.0
+     */
+    void createOffer(String tomatoId)
+
+    /**
+     *
+     * @param projectTitle
+     * @param projectDescription
+     * @param customer
+     * @param projectManager
+     * @param productItems
+     * @since 1.0.0
+     */
+    void createNewOffer(Quotation quotation)
```

Comment:
the parameters here do not fit the method signature

Reply:
Thanks for pointing it out, I changed the signature back so the description fits the method. Or do you think it's better to transfer the Offer DTO here instead of the fields?